megablocks/__init__.py

```diff
@@ -1,2 +1,12 @@
-import megablocks.layers.dmoe
-import megablocks.layers.moe
+# Copyright 2024 MosaicML MegaBlocks authors
```

Comment:
Do you have a plan to include the [header file](https://github.com/mosaicml/llm-foundry/blob/main/.pre-commit/FILE_HEADER) either in this PR or next? This will automatically add the copyright header when running the pre-commit.

Reply:
I'm doing this in a follow up PR.